src/main/java/io/snyk/snyk_maven_plugin/AbstractSnykMojo.java

```diff
@@ -43,7 +44,8 @@ public int executeCommand() throws MojoExecutionException {
                 getExecutable().getAbsolutePath(),
                 getCommand(),
                 Optional.ofNullable(apiToken),
-                args
+                args,
+                MessageUtils.isColorEnabled()
```

Comment:
Haven't found a doc describing what is being considered for `isColorEnabled` - whether it's only system capabilities or also flags/options